src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -101,7 +105,71 @@ public void Start(ApplicationData applicationData)
                 columnWidthsSum--;
             }
 
-            win.Add(new Label(GetPaddedString(gridHeaders, columnWidths, offset + offset - 1)));
+            var filterLabel = new Label("Filter")
+            {
+                X = 2
+            };
+
+            // 7 here is to reserve space for the Apply button
+            var filterFieldWidth = usableWidth - filterLabel.Text.Length - 7;
```

Comment:
Ideally the button name should be localized. So I suggest to use a const now and length() method instead of 7.

Reply:
Will update.  We should eventually move all text to a resx.